lib/v2/screens/explore_screens/vote_screens/proposal_details/components/proposal_details_header.dart

```diff
@@ -1,41 +1,69 @@
 import 'package:cached_network_image/cached_network_image.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/domain-shared/ui_constants.dart';
 import 'package:seeds/v2/images/vote/proposal_category.dart';
-import 'package:seeds/v2/design/app_theme.dart';
 import 'package:seeds/v2/screens/explore_screens/vote_screens/proposal_details/interactor/viewmodels/bloc.dart';
 
 class ProposalDetailsHeader extends StatelessWidget {
   const ProposalDetailsHeader({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
+    double width = MediaQuery.of(context).size.width;
     return BlocBuilder<ProposalDetailsBloc, ProposalDetailsState>(
       buildWhen: (previous, current) => previous.currentIndex != current.currentIndex,
       builder: (context, state) {
         return Container(
-          height: 280,
+          clipBehavior: Clip.hardEdge,
+          decoration: const BoxDecoration(
+              borderRadius: BorderRadius.only(
+                  bottomLeft: Radius.circular(defaultCardBorderRadius),
+                  bottomRight: Radius.circular(defaultCardBorderRadius))),
+          height: width,
           child: Stack(
             fit: StackFit.expand,
             children: [
               FittedBox(
-                fit: BoxFit.fitWidth,
+                clipBehavior: Clip.hardEdge,
+                fit: BoxFit.cover,
```

Comment:
ohh.. this was the real fix

Reply:
yep, I play with the settings for 2h until I got it right